packages/mjs/loader/index.js

```diff
@@ -31,21 +31,28 @@
   })();
 
   var config = __CONFIG__;
+  var analyticsIsNew =
+    !!~config.analyticsjs_version.indexOf('2.0') ||
+    !!~config.analyticsjs_version.indexOf('3.');
```

Comment:
wont it break if its 1.x ?

Reply:
no, this loader uses just for new MJS and Analytics